### DISCUSSION TITLE
EditableGeoJsonLayer updateTriggers not working as expected

### DISCUSSION THREAD

**Discussion by erikhagendorn** [visgl/deck.gl-community#113] [General]
Hi everyone,

I've run into an interesting situation with the EditableGeoJsonLayer that I'm hoping to get some insight on. I've been trying to use updateTriggers to update the fillColor of my layer, but it doesn't seem to be working as I'd expect.

Here's the interesting part: when I swap out EditableGeoJsonLayer for the standard GeoJsonLayer, everything works perfectly. Same code otherwise.

I've tried various approaches to get this working with EditableGeoJsonLayer, but no luck so far. I'm starting to wonder if this is a known limitation or if there's a different approach I should be taking with EditableGeoJsonLayer.

Has anyone else encountered this? Are updateTriggers supposed to work differently with EditableGeoJsonLayer? Or is there perhaps a bug that I should be reporting?

If this is expected behavior, I'd love to know the recommended way to update properties like fillColor for EditableGeoJsonLayer.

Any insights or suggestions would be greatly appreciated!

Thanks in advance for your help!

**Comment by ibgreen** [comment]
Maybe worth looking at the code or single stepping throuhg to check if the `updateTriggers` props are being pass through to the underlying layers? Also there has been some changes in the naming of certain props/triggers over the years, so it could be that things simply have gotten out of sync as editable-layers maintainers moved on?

**Comment by erikhagendorn** [comment]
Makes sense, can do that. Thanks.
